src/ci/docker/scripts/android-start-emulator.sh

```diff
@@ -11,6 +11,6 @@ export SHELL=/bin/bash
 # classic engine the emulator starts with the current date and the tests run
 # fine. If another image is used, this need to be evaluated again.
 nohup nohup emulator @armeabi-v7a-18 \
-    -engine classic -no-window -partition-size 2047 0<&- &>/dev/null &
+    -engine classic -no-window -partition-size 2047 0<&- 2>&1 /dev/null &
```

Comment:
This doesn't look right. Did you mean to redirect both?
```suggestion
    -engine classic -no-window -partition-size 2047 0<&- >/dev/null 2>&1 &
```
If so, why did you change it? It was the same before but shorter.

Reply:
You are right, it is what I meant. I changed it because POSIX sh does not support, &>, only bash.